test/cfg/qt.cpp

```diff
@@ -573,4 +573,14 @@ namespace {
     // simplifyQtSignalsSlots2
     namespace Foo { class Bar; }
     class Foo::Bar : public QObject { private slots: };
+
+    // Q_PROPERTY with templates inducing a ',' should not produce preprocessorErrorDirective
+    class AssocProperty : public QObject {
+    public:
+        Q_PROPERTY(QHash<QString, int> hash READ hash WRITE setHash)
+        QHash<QString, int> hash() const;
+        void setHash(const QHash<QString, int>& hash);
+    private:
+        QHash<QString, int> m_hash;
```

Comment:
@chrchr-github: I guess for the test this block is superfluous, please let me know if I shall remove it.

Reply:
I just checked without this block, and changing Q_PROPERTY(...) back to Q_PROPERTY(X) leads to a fail, and back to Q_PROPERTY(...) passes the test.